modules/UI/authentication/AuthHandler.js

```diff
@@ -1,25 +1,27 @@
-/* global APP, config, JitsiMeetJS, Promise */
+// @flow
 
 import Logger from 'jitsi-meet-logger';
 
 import { openConnection } from '../../../connection';
-import { setJWT } from '../../../react/features/base/jwt';
 import {
-    JitsiConnectionErrors
-} from '../../../react/features/base/lib-jitsi-meet';
+    openWaitForOwnerDialog
+} from '../../../react/features/authentication/actions.web';
+import {
+    isTokenAuthEnabled,
+    getTokenAuthUrl
+} from '../../../react/features/authentication/functions';
+import { setJWT } from '../../../react/features/base/jwt';
+import JitsiMeetJS from '../../../react/features/base/lib-jitsi-meet';
 import UIUtil from '../util/UIUtil';
 
 import LoginDialog from './LoginDialog';
 
-const logger = Logger.getLogger(__filename);
-
 let externalAuthWindow;
 let authRequiredDialog;
```

Comment:
No need to have this dialog reference anymore since we are creating it with React.